src/components/Modal/ModalLayout/ModalLayout.tsx

```diff
@@ -36,7 +37,7 @@ function ModalLayout({ children, handleModalClose }: ModalLayoutProps) {
   }, [handleModalClose]);
 
   return createPortal(
-    <div onClick={modalOutsideClick} ref={modalOutsideRef} className={styles.root}>
+    <div onClick={modalOutsideClick} ref={modalOutsideRef} className={!alarmModal ? styles.root : styles.alarmModalCss}>
```

Comment:
```suggestion
  <div onClick={modalOutsideClick} ref={modalOutsideRef} className={`${styles.root} ${alarmModal && styles.alarmModal}`}>
      <div>{children}</div>
    </div>
```
css
```
.alarmModal {
  background: none;
}
```
기존에 작성된 부분은 겹치는 내용이 많고 background: none만 있어도 의도한대로 동작하는 것 같습니다.